control

```diff
@@ -9,9 +9,9 @@ XB-Eula: eula-ni-standard,
+XB-AlwaysRemoves: ni-scan-engine-veristand-2019-support, ni-scan-engine-veristand-2020-support, ni-synchronization-veristand-2019-support, ni-synchronization-veristand-2020-support, ni-engine-simulatio-veristand-2019-support, ni-engine-simulation-veristand-2020-support, ni-slsc12201-veristand-2019-support, ni-slsc12201-veristand-2020-support, ni-slsc-eds-veristand-2019-support, ni-slsc-eds-veristand-2020-support, ni-routing-and-faulting-veristand-2019-support, ni-routing-and-faulting-veristand-2020-support, ni-slsc-switch-veristand-2019-support, ni-slsc-switch-veristand-2020-support, ni-embedded-data-logger-veristand-2020-support, ni-routing-and-faulting-veristand-2019-labview-support, ni-routing-and-faulting-veristand-2020-labview-support, ni-slsc-switch-veristand-2019-labview-support, ni-slsc-switch-veristand-2020-labview-support, ni-routing-and-faulting-veristand-2019-labview-examples, ni-routing-and-faulting-veristand-2020-labview-examples, ni-instrument-addon-veristand-2019-support, ni-instrument-addon-veristand-2020-support, ni-instrument-addon-veristand-2019-labview-support, ni-instrument-addon-veristand-2020-labview-support, ni-fpga-addon-veristand-2019-support, ni-fpga-addon-veristand-2020-support, ni-fpga-addon-veristand-2019-labview-support, ni-fpga-addon-veristand-2020-labview-support, ni-fpga-addon-veristand-2019-labview-support-examples, ni-fpga-addon-veristand-2020-labview-support-examples, ni-data-sharing-framework-veristand-2019-support, ni-data-sharing-framework-veristand-2020-support, ni-data-sharing-framework-veristand-2019-labview-examples, ni-data-sharing-framework-veristand-2020-labview-examples, ni-data-sharing-framework-veristand-2019-labview-support, ni-data-sharing-framework-veristand-2020-labview-support, ni-data-sharing-framework-plugins-veristand-2019-support, ni-data-sharing-framework-plugins-veristand-2020-support
```

Comment:
You deleted an 'n' from ni-engine-simulation-veristand-2019-support. Add this back.

Reply:
Fixed the missing letter.